.gitignore

```diff
@@ -77,6 +77,7 @@ target/
 
 # VSCode
 .vscode
+.idea/
```

Comment:
This is not the purpose of the PR, but I understand the usefulness of the line (no change requested).